setup.cfg

```diff
@@ -13,3 +13,6 @@ include_trailing_comma = True
 force_grid_wrap = 0
 use_parentheses = True
 line_length = 88
+
+[codespell]
+ignore-words-list=ORE,ore
```

Comment:
Alternatively we can create a file with one word per line, and use ignore-file=FILE_NAME.